tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml

```diff
@@ -87,6 +87,24 @@ exon sequences and transcript structures.
 is used as an input for ``ribotaper part 2``.
 *Annotation_path*
 is used as an input for ``ribotaper part 3``.
+
+Important notes
```

Comment:
I don't think we need this and it should probably not in the tool XML file. This is information for an admin and should be in a repository README.rst file.